src/tunnels/tunnel_script_renderer/templates/ssh_config_template.conf

```diff
@@ -0,0 +1,6 @@
+Host ${host_friendly_name}
+    HostName ${server_domain}
+    Port ${reverse_port} ${reverse_server_ssh_port}
+    Compression yes
```

Comment:
Will there be any loss of performance in the connection between SSH servers if we set `Compression`?